src/aap_eda/core/migrations/0023_add_default_organization.py

```diff
@@ -0,0 +1,76 @@
+from django.conf import settings
+from django.db import migrations
+from django.utils import timezone
+
+
+def create_default_org(apps, schema_editor):
+    Organization = apps.get_model("core", "Organization")  # noqa: N806
+
+    db_alias = schema_editor.connection.alias
+
+    now = timezone.now()
+    Organization.objects.using(db_alias).create(
```

Comment:
might be good to call `get_or_create` just in case you or I want to reverse and re-apply the migration for testing.